jwtauth.go

```diff
@@ -78,30 +100,45 @@ func (ja *JwtAuth) Handle(paramAliases ...string) func(chi.Handler) chi.Handler
 
 			// Token is required, cya
 			if tokenStr == "" {
-				err = errUnauthorized
+				err = ErrUnauthorized
 			}
 
 			// Verify the token
 			token, err := ja.Decode(tokenStr)
 			if err != nil || !token.Valid || token.Method != ja.signer {
-				http.Error(w, errUnauthorized.Error(), 401)
+				switch err.Error() {
+				case "token is expired":
```

Comment:
I agree, but the upstream jwt package makes a little bit of a mess with their error handling, and this makes it work. it passes tests for now, we can revisit this as they release their next version (3.0) which is coming soon